docs/real-time/snippets/signalr/SignalR.ApiService/Program.cs

```diff
@@ -19,41 +28,38 @@
             .BuildServiceManager();
     });
 }
-else
-{
-    builder.Services.AddSignalR()
-                    .AddNamedAzureSignalR("signalr");
-}
-
 var app = builder.Build();
 
 app.UseExceptionHandler();
 
+var options = new JsonSerializerOptions(JsonSerializerDefaults.Web)
+{
+    DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
+};
+
 if (isServerlessMode)
 {
-    app.MapPost("/negotiate", async (ServiceManager serviceManager, string? userId) =>
+    app.MapPost("/chathub/negotiate", async (ServiceManager sm, ILogger<Program> logger) =>
     {
-        var healthy = await serviceManager.IsServiceHealthy(CancellationToken.None);
-        if (healthy is false)
-        {
-            return Results.Problem("SignalR service is not healthy.");
-        }
+        var hubContext = await sm.CreateHubContextAsync("chathub", CancellationToken.None);
```

Comment:
this creation is a little bit heavy, could we move the creation out. There is actually no need to DI ServiceManager, we could also remove that part. We simplified the aspire sample a little bit and hope it is clearer now https://github.com/dotnet/aspire/blob/main/playground/signalr/SignalRServerlessWeb/Program.cs